src/app/services/logger/reducers/index.ts

```diff
@@ -0,0 +1,20 @@
+import {
```

Comment:
needs to go to the store dir

Reply:
There is already an index.ts in the store directory

Reply:
DOne